src/main/java/teammates/common/datatransfer/FeedbackTextResponseDetails.java

```diff
@@ -15,28 +17,28 @@ public FeedbackTextResponseDetails() {
     
     public FeedbackTextResponseDetails(String answer) {
         super(FeedbackQuestionType.TEXT);
-        this.answer = answer;
+        this.answer = answer == null ? null : Sanitizer.sanitizeForRichText(answer);
```

Comment:
This bit of code seem to repeat multiple times. Abstract into a small utility method?